### PR TITLE
[MRG] medor_website_p_s -> medor_custom

### PR DESCRIPTION
**Pull request by remytms**
Merge medor_website_product_subscription into medor_custom.

Deployment information:

**Warning** Uninstalling `medor_website_product_subscription` should be
done before applying this commit. Removing this module will also remove
`medor_custom`. If you want to avoid removing `medor_custom`, first
manually remove `medor_website_product_subscription` from dependence
list of `medor_custom` then update `medor_custom`. After that, uninstall
`medor_website_product_subscription` and finally apply this commit, and
update `medor_custom`.

1. Uninstall `medor_website_product_subscription` (this will also
   uninstall `medor_custom`, see warning above).
2. Apply this commit.
3. Install `medor_custom`.

### CODE REVIEW

File: medor_custom/controllers/user.py [hunk 1]
```diff
@@ -36,14 +33,12 @@ def user_form(self, **kw):
...
-                })
+                company.write({"name": request.params["company_name"]})
                 try:
-                    user.vat = request.params['vat']
+                    user.vat = request.params["vat"]
                 except ValidationError as err:
-                    request.params['error'] = err.name
-                if request.params.get('invoice_address', False):
+                    request.params["error"] = err.name
+                if request.params.get("invoice_address", False):
```
Comment (robinkeunen) [inline medor_custom/controllers/user.py:41]: Try to do formatting improvements in separate commits, it makes the review more readable.

**Review by robinkeunen — APPROVED** [review]
It looks good but next time, blacken the files in a separate commit so I can focus my review on things that matter. :-)

File: medor_custom/controllers/user.py (hunk 1 above)
Comment (remytms) [reply medor_custom/controllers/user.py:41]: You're right, sorry. Thanks for reviewing. :)
